Q-And-A/Q-And-A/ViewControllers/AnswerViewController.swift

```diff
@@ -0,0 +1,55 @@
+//
+//  AnswerViewController.swift
+//  Q-And-A
+//
+//  Created by Angel Buenrostro on 1/15/19.
+//  Copyright © 2019 Angel Buenrostro. All rights reserved.
+//
+
+import UIKit
+
+class AnswerViewController: UIViewController {
+    
+    var questionController: QuestionController? = nil
+    
+    var question: Question? = nil
+
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var questionLabel: UILabel!
+    @IBOutlet weak var answererLabel: UITextField!
+    @IBOutlet weak var answerView: UITextView!
+    @IBAction func submitAnswerTapped(_ sender: Any) {
+        guard let answererLabel = answererLabel?.text else { return }
+        guard let answerView = answerView?.text else { return }
+        
+        questionController?.Update(question: question!, answer: answerView, answerer: answererLabel)
+        self.navigationController?.popViewController(animated: true)
+    }
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        // Do any additional setup after loading the view.
+    }
+    
+    func updateViews(){
+        let answer = self.question?.answer
+        let answerer = self.question?.answerer
+        let name = self.question?.asker
+        let question = self.question?.question
```

Comment:
Before assigning `question`'s values to the labels, you need to unwrap it. And also don't want to run this code if view isn't loaded, because at that time the label's is going to be nil and app crash. We could check `isViewLoaded` value and then run it
```suggestion
        guard isViewLoaded else { return }
        guard let question = question else { return }
```